XGBoost/XGBoost (1).py

```diff
@@ -0,0 +1,96 @@
+# XGBoost 
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import IPython
+from IPython import get_ipython
+
+
+# Consider the following data.
+class XGB:
+    def __init__(self):
+        age = [9, 23, 24, 27, 45, 56, 62, 71, 76, 82, 84, 95, 79, 29, 34, 35, 49]

```

Comment:
You have used fixed data,please remove it and make it work for a general case

Reply:
done